lib/measures/tbd/measure.rb

```diff
@@ -253,6 +253,14 @@ def arguments(model = nil)
     kiva_force.setDefaultValue(false)
     args << kiva_force
 
+    arg = "reset_kiva"
+    dsc = "Purges KIVA entries from model before generating Kiva inputs."
+    reset_kiva = OpenStudio::Measure::OSArgument.makeBoolArgument(arg, false)
+    reset_kiva.setDisplayName("Purge existing KIVA inputs")
+    reset_kiva.setDescription(dsc)
+    reset_kiva.setDefaultValue(false)
+    args << reset_kiva
```

Comment:
New measure boolean argument: purge existing KIVA objects in model (unchecked by default). This option can be activated in isolation from all other TBD options.